src/cli/init.rs

```diff
@@ -141,6 +247,17 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     Ok(())
 }
 
+fn get_dir_name(path: &Path) -> miette::Result<String> {
+    Ok(path
+        .file_name()
+        .ok_or(miette::miette!(
+            "Cannot get file or directory name from the path: {}",
+            path.to_string_lossy()
+        ))?
+        .to_string_lossy()
+        .to_string())
+}
```

Comment:
I would expect to get a directory name from this function but according to the implementation, it returns the filename.

Reply:
Changed name to `get_name_from_dir`